packaging/pep517_backend/_backend.py

```diff
@@ -73,7 +73,7 @@
 CYTHON_TRACING_CONFIG_SETTING = 'with-cython-tracing'
 """Config setting name toggle to include line tracing to C-exts."""
 
-CYTHON_TRACING_ENV_VAR = 'PROPCACHECYTHON_TRACING'
+CYTHON_TRACING_ENV_VAR = 'PROPCACHE_CYTHON_TRACING'
```

Comment:
We should eventually make it read this from `pyproject.toml`, perhaps. So the backend could be copied across projects w/o us needing to adapt it..